xlive/H2MOD/Modules/Shell/Config.cpp

```diff
@@ -494,6 +496,7 @@ void SaveH2Config()
 			CONFIG_SET(&ini, "use_d3d9on12", &g_rasterizer_dx9on12_enabled);
 			CONFIG_SET(&ini, "disable_amd_or_ati_patches", &g_rasterizer_dx9_driver_globals.disable_amd_or_ati_patches);
 			CONFIG_SET(&ini, "use_vsync", &H2Config_use_vsync);
+			CONFIG_SET(&ini, "splitscreen_mode", std::to_string(H2Config_split_mode).c_str());
```

Comment:
why is this being converted to a string?
Shouldn't we just cast this to an integer and then use the set functionality for that?